.github/workflows/build-test.yml

```diff
@@ -210,3 +214,33 @@ jobs:
                 CLOUD_PROVIDER: ${{ matrix.cloud }}
                 TARGET_DOCKER_TEST_IMAGE: ${{ matrix.image }}
               run: ./ci/test.sh
+
+    display_code_coverage:
+        needs: test-mac-required
+        name: Display code coverage
+        runs-on: macos-latest
```

Comment:
do we need macos? cannot we use e.g. ubuntu and then do not install bash?

Reply:
I stayed with macos ... but only running command without step of script and bash installing